### PR TITLE
[Not approve twice the PR]

### PR DESCRIPTION
**Pull request by emravera**

### CODE REVIEW

File: lib/routes/run.js [hunk 1]
```diff
@@ -159,15 +159,21 @@ module.exports = function* (next) {
...
     const reviewReq = {
       repo: base.repo.full_name,
       pr: number,
       event: APPROVE_PR,
       body: 'Yey!! this PR doesn\'t contain linter errors',
     };
+
     // If a review was found, sent a submit review for that specific review
-    if (review) {
+    if (review && review.review_id) {
```
Comment (davidverme) [inline lib/routes/run.js]: Shouldn't it be 
`if (review && review.id)` ?

File: lib/routes/run.js [hunk 2]
```diff
@@ -159,15 +159,21 @@ module.exports = function* (next) {
 
   // If no errors Approve the PR automatically
   if (!result && reportCss.length === 0) {
-    this.body = `No problems were found for PR ${number}`;
+    this.body = `No problems were found for PR https://github.com/${base.repo.full_name}/pull/${number}`;
+    // If already approved return
+    if (review && review.state === 'APPROVED') {
```
Comment (emaperez) [inline lib/routes/run.js]: can you reaname review variable . A better name can be botLastReview

File: lib/routes/run.js (hunk 1 above)
Comment (emravera) [reply lib/routes/run.js]: The API call we are using needs that variable to no conflict with the id of the request. So I will left like this. Thanks for the comment.

File: lib/routes/run.js (hunk 2 above)
Comment (emravera) [reply lib/routes/run.js]: 1e0a8a9

File: lib/routes/run.js (hunk 1 above)
Comment (emravera) [reply lib/routes/run.js]: 1e0a8a9

**Pull request merged**